source/authentication/email-password.txt

```diff
@@ -93,6 +94,22 @@ Configuration
 
 User Confirmation
 -----------------
+User confirmation occurs when a user wants to create a new Email/password 
```

Comment:
I wonder if we want to reframe this in terms of what the developer must do here? i.e. something like:

> Before an email/password user can authenticate, your app must register and 
> confirm the user. Registering an email/password user creates a new user object. 
> App Services then requires this user to confirm the creation of this account. You
> can select one of three methods to confirm a new user account:

Reply:
Slightly reworded because AAS doesn't necessarily require the _user_ to confirm their account. 

> Before an email/password user can authenticate, your app must register and
confirm the user's new account. Registering an email/password user creates a 
new user object. App Services then requires confirmation of the account. You can 
select one of three methods to confirm a new user account: